outwatch/src/main/scala/outwatch/util/Store.scala

```diff
@@ -86,15 +86,14 @@ object Store {
       }).get
     }
 
-    val sub = subject.subscribe()
```

Comment:
I forgot about this - I was trying to figure out why the initial action passed is not actually dispatched on the store that's created. I think it has something to do with `replay(1).refCount`. These methods are marked as side effecting, and at the time of their calls there will be no subscribers. I was not able to get this to work tho

Reply:
Oh yeah, the store is sadly not perfect yet (also see https://github.com/OutWatch/outwatch/issues/289). Ideas and PRs are definitely welcome.

Can you open an issue with your example?

Reply:
Yeah this was just my first step trying to peer into why this is happening. My issue I think is the same as #289, I brought it up in the gitter:

https://gitter.im/OutWatch/Lobby?at=5d39c13b9114f065e2c89953